src/app/technology/middleware.js

```diff
@@ -7,36 +7,50 @@ import {
 } from '../../common';
 
 const areRequiredParamsPresent = (req, res, next) => {
-  const requiredParams = ['name', 'category'];
+  const requiredParams = ['name', 'category', 'index'];
   return requiredParamsValidator(req.body, requiredParams, next);
 };
 
 const areTypesValid = (req, res, next) => {
-  const { name, category } = req.body;
-  const stringParams = { name, category };
+  const { name, category, index } = req.body;
+  const stringParams = { name, category, index };
   return typeValidator(stringParams, 'string', next);
 };
 
+const isIndexFloat = (req, res, next) => {
+  const { index } = req.body;
+  const floatRegex = /^[-+]?[0-9]+\.[0-9]+$/;
+  if (index && index.match(floatRegex)) return next();
+
+  const error = new Error('index must be of type: float');
```

Comment:
Can you make this `[index]` instead of `index`?

It'll help keep our `type` error messages consistent.

Reply:
Of course!